Week2.py

```diff
@@ -0,0 +1,90 @@
+
+import unittest
+
+def find_triplets(string, target_sum):
+
+    ### this code works for any target sum value ####
+
+    if(string is None or ''):
+        return None
+
+    
+    size=len(string)
+    arr=[0]*size
+
+    ### converting input string into list of integer values ###
+    arr=map(int, string.split())
+    
+    arr.sort()
```

Comment:
this should be done after the size check.  While it won't make much difference since the array is small, there is no need to sort call sort if we are just going to return None for small arrays